evalml/tests/pipeline_tests/test_pipelines.py

```diff
@@ -230,6 +258,38 @@ def test_make_pipeline_no_column_names(problem_type):
                 assert pipeline.component_graph == [DropNullColumns, Imputer, OneHotEncoder, estimator_class]
 
 
+@pytest.mark.parametrize("input_type", ["pd", "ww"])
+@pytest.mark.parametrize("problem_type", [ProblemTypes.BINARY, ProblemTypes.MULTICLASS, ProblemTypes.REGRESSION])
+def test_make_pipeline_text_columns(input_type, problem_type):
+    X = pd.DataFrame({"numerical": [1, 2, 3, 1, 2],
+                      "categorical": ["a", "b", "a", "c", "c"],
+                      "text": ["string one", "another", "text for a column", "text string", "hello world"]})
+    y = pd.Series([0, 0, 1, 1, 0])
+    if input_type == 'ww':
+        X = ww.DataTable(X)
+        y = ww.DataColumn(y)
+    estimators = get_estimators(problem_type=problem_type)
+
+    if problem_type == ProblemTypes.BINARY:
+        pipeline_class = BinaryClassificationPipeline
+    elif problem_type == ProblemTypes.MULTICLASS:
+        y = pd.Series([0, 2, 1, 2])
+        pipeline_class = MulticlassClassificationPipeline
+    elif problem_type == ProblemTypes.REGRESSION:
+        pipeline_class = RegressionPipeline
+    for estimator_class in estimators:
+        for problem_type in estimator_class.supported_problem_types:
+            pipeline = make_pipeline(X, y, estimator_class, problem_type, text_columns=['text'])
+            assert isinstance(pipeline, type(pipeline_class))
+            assert pipeline.custom_hyperparameters is None
+            if estimator_class.model_family == ModelFamily.LINEAR_MODEL:
+                assert pipeline.component_graph == [Imputer, TextFeaturizer, OneHotEncoder, StandardScaler, estimator_class]
+            elif estimator_class.model_family == ModelFamily.CATBOOST:
+                assert pipeline.component_graph == [Imputer, TextFeaturizer, estimator_class]
+            else:
+                assert pipeline.component_graph == [Imputer, TextFeaturizer, OneHotEncoder, estimator_class]
```

Comment:
Yes!!!! One step closer to having the `TextFeaturizer` get text columns directly from the woodwork datatable
![hooray](https://user-images.githubusercontent.com/235870/98418756-9e95f680-2051-11eb-8b90-fbd834cca287.gif)